packages/ui/components/Select.tsx

```diff
@@ -0,0 +1,99 @@
+import * as Radix from '@radix-ui/react-select'
+import type {ElementRef, ReactNode} from 'react'
+import {forwardRef} from 'react'
+import {twMerge} from 'tailwind-merge'
+import {CheckIcon, ChevronDownIcon} from '../icons'
+
+export {Select, SelectGroup, SelectLabel} from '@radix-ui/react-select'
+
+type TriggerRef = ElementRef<typeof Radix.SelectTrigger>
+type TriggerProps = Radix.SelectTriggerProps & {
+  placeholder?: ReactNode
+}
+
+export const SelectTrigger = forwardRef<TriggerRef, TriggerProps>(
+  function SelectTrigger(props, ref) {
+    const {children, className, placeholder, ...triggerProps} = props
+    return (
+      <Radix.SelectTrigger
+        ref={ref}
+        className={twMerge(
+          'inline-flex h-8 w-full items-center justify-between gap-2 rounded-lg bg-venice-black-500 px-4 font-mono text-xs text-offwhite ring-1 ring-inset ring-venice-black-300 focus:outline-none focus-visible:ring-venice-green disabled:cursor-default disabled:opacity-50 data-[placeholder]:text-venice-gray-muted',
+          className,
+        )}
+        {...triggerProps}>
+        <Radix.SelectValue placeholder={placeholder} />
+        <Radix.SelectIcon className="shrink-0">
+          <ChevronDownIcon className="h-2.5 w-2.5 fill-current" />
+        </Radix.SelectIcon>
+      </Radix.SelectTrigger>
+    )
+  },
+)
+
+type ContentRef = ElementRef<typeof Radix.Content>
+type ContentProps = Radix.SelectContentProps
+
+// TODO use SelectScrollUpButton and SelectScrollDownButton
+export const SelectContent = forwardRef<ContentRef, ContentProps>(
+  function SelectContent(props, ref) {
+    const {children, className, ...contentProps} = props
+    return (
+      <Radix.SelectPortal>
+        <Radix.SelectContent
+          ref={ref}
+          className={twMerge(
+            'min-w-[8rem] animate-slide-down overflow-hidden rounded-lg border border-venice-black-300 bg-venice-black-500 px-1 py-2 text-sm drop-shadow-md animate-in fade-in-80',
```

Comment:
The only UX nit I have here is it might not be apparent that there are items needing to scroll into view unless you're familiar with the schema. Could we give this a larger default height or handle this in another way?

e.g. with `Resource` selected, tapping the select list only shows
<img width="185" alt="Screen Shot 2023-02-07 at 8 24 33 AM" src="https://user-images.githubusercontent.com/317671/217303420-2ed39b2f-1eaf-4baa-912f-d1b97c576edd.png">

when really there are more items to display if the user scrolls up
<img width="185" alt="Screen Shot 2023-02-07 at 8 25 41 AM" src="https://user-images.githubusercontent.com/317671/217303582-dc05e33d-7325-49e2-a299-1bf18931e23d.png">

Reply:
It's not a nit at all! It's confusing. That's the UX improvement I mentioned (during stand-up) for `Select` element that I plan to do.

The pop-up has collision detection and limits the height when it hits the top of the pop-up boundary (the document in this case). We will show an indicator (arrow) that there're more items.